### ISSUE TITLE
Train from scratch, I got this problem!

### ISSUE THREAD

**Issue by RobotIntelligence** [roytseng-tw/Detectron.pytorch#137]
RuntimeWarning: invalid value encountered in greater_equal (x_ctr < im_info[1]) & (y_ctr < im_info[0]))[0]

**Comment by GYxiaOH** [comment]
you can decrease learning rate(or init learnning rate in warm up) or dealy warm up iterations
